tests/_test_msui/test_wms_control.py

```diff
@@ -495,9 +495,11 @@ class TestWMSControlWidgetSetupSimple:
         <Extent name="ELEVATION" default="900.0"> 500.0,600.0,700.0,900.0 </Extent>"""
 
     @pytest.fixture(autouse=True)
-    def setup(self, qtbot):
+    def setup(self, tmp_path, qtbot):
+        wc.WMS_SERVICE_CACHE = {}
```

Comment:
Having to reset this here is a good indicator that this variable shouldn't be global.